diff --git a/yabause/src/vdp2.cpp b/yabause/src/vdp2.cpp
index 1fce632c..7a4d84f8 100644
--- a/yabause/src/vdp2.cpp
+++ b/yabause/src/vdp2.cpp
@@ -1093,7 +1093,8 @@ static void FPSDisplay(void)
   static u64 fpsticks;
   //yprintf("%02d/%02d FPS skip=%d vdp1=%02d", fps, yabsys.IsPal ? 50 : 60, show_skipped_frame, show_vdp1_frame);
 #if 1 // FPS only
-   OSDPushMessage(OSDMSG_FPS, 1, "%02d/%02d FPS skip=%d vdp1=%02d", fps, yabsys.IsPal ? 50 : 60, show_skipped_frame, show_vdp1_frame);
+   if (OSDIsVisible(OSDMSG_FPS) > 0)
+      OSDPushMessage(OSDMSG_FPS, 1, "%02d/%02d FPS skip=%d vdp1=%02d", fps, yabsys.IsPal ? 50 : 60, show_skipped_frame, show_vdp1_frame);
    //printf("\033[%d;%dH %02d/%02d FPS skip=%d vdp1=%02d \n", 0, 0, fps, yabsys.IsPal ? 50 : 60, show_skipped_frame, show_vdp1_frame);
 #else
   FILE * fp = NULL;
