--- a/yabause/CMake/Packages/external_libchdr.cmake	2026-01-30 03:13:35.268749454 +0000
+++ b/yabause/CMake/Packages/external_libchdr.cmake	2026-01-30 03:13:56.112711985 +0000
@@ -29,9 +29,9 @@ endif()
 
 ExternalProject_Add(
   libchdr
-  GIT_REPOSITORY "https://github.com/devmiyax/libchdr.git"
-  GIT_TAG "074ff1614f2a685f2b5a95b0e788bff6297d5680"
-  #PATCH_COMMAND  git apply "${CMAKE_SOURCE_DIR}/CMake/Packages/libchdr.patch"
+  URL "https://github.com/devmiyax/libchdr/archive/074ff1614f2a685f2b5a95b0e788bff6297d5680.tar.gz"
+  DOWNLOAD_NO_PROGRESS 1
+  UPDATE_COMMAND ""
   CMAKE_ARGS  -DCMAKE_INSTALL_PREFIX=${CMAKE_CURRENT_BINARY_DIR}/libchdr 
               -DCMAKE_BUILD_TYPE:STRING=Release 
               -DCMAKE_CXX_FLAGS=${CMAKE_CXX_FLAGS}
